src/test/merkle_tests.cpp

```diff
@@ -268,6 +268,8 @@ BOOST_FIXTURE_TEST_SUITE(merkle_tests, BasicTestingSetup)
 
 BOOST_AUTO_TEST_CASE(tree_test_vectors)
 {
+    libsnark::default_r1cs_ppzksnark_pp::init_public_params();
+
```

Comment:
IIRC there's a function that calls the right `init_public_params()` (there are multiple of them) for the things it's actually configured to use. Is this that function?